starport/pkg/entrywriter/entrywriter_test.go

```diff
@@ -0,0 +1,25 @@
+package entrywriter_test
+
+import (
+	"github.com/stretchr/testify/require"
+	"github.com/tendermint/starport/starport/pkg/entrywriter"
+	"io"
+	"testing"
+)
+
+func TestWrite(t *testing.T) {
```

Comment:
Can we use a table test so we can cover all error cases?

Reply:
I added a case for when a write error occurs but TBH the tests are small for a TDT being beneficial since there is no output to compare